worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -341,6 +353,8 @@ class PushDataHandler(val workerSource: WorkerSource) extends BaseMessageHandler
           }
         }
       })
+      pushData.body().retain()
```

Comment:
I don't see the reason to call `retain()` here, could you explain a bit about the reason? L255 already called `retain()`

Reply:
yes, you're right we don't need it here, I missed it.